ci/provisioning/post_provision_config_nodes_EL_7.sh

```diff
@@ -114,12 +114,26 @@ post_provision_config_nodes() {
     fi
     rm -f /etc/profile.d/openmpi.sh
     rm -f /tmp/daos_control.log
-    time dnf -y install $LSB_RELEASE
 
+    if ! rpm -q "$LSB_RELEASE"; then
+        time dnf -y install "$LSB_RELEASE"
+    fi
+
+    n=3
+    rc=0
+    # shellcheck disable=SC2001
     # shellcheck disable=SC2086
-    if [ -n "$INST_RPMS" ] &&
-       ! time dnf -y install $INST_RPMS; then
-        rc=${PIPESTATUS[0]}
+    if ! rpm -q "$(echo "$INST_RPMS" |
+                   sed -e 's/--exclude [^ ]*//'
+                       -e 's/[^ ]*-daos-[0-9][0-9]*//g')"; then
```

Comment:
(lint) This flag is used as a command name. Bad line break or missing [ .. ]? [SC2215]